mvn failures in TCK buckets are not reported properly

If a failure occurs during the `mvn build` phase of a TCK bucket, a suitable error message is **not** returned. For example, given a bad dependency in the `pom.xml`, such as:
```
<dependency>
    <groupId>a.dumb.dependency</groupId>
    <artifactId>i-should-break</artifactId> 
    <version>2.0</version>
</dependency>
```
Then something like the following is returned in the `output.txt` file, as shown in Defect 283349:
```
junit.framework.AssertionFailedError: 2021-04-22-22:22:39:064 No TCK test JUnit result files were found in the results directory which suggests the TCK tests did not run
Errors found in mvnOutput were:
### maven test output:

at componenttest.topology.utils.MvnUtils.findJunitResultFiles(MvnUtils.java:181)
at componenttest.topology.utils.MvnUtils.postProcessTestResults(MvnUtils.java:483)
at componenttest.topology.utils.MvnUtils.runCleanTest(MvnUtils.java:636)
at componenttest.topology.utils.MvnUtils.runTCKMvnCmd(MvnUtils.java:142)
at componenttest.topology.utils.MvnUtils.runTCKMvnCmd(MvnUtils.java:112)
at org.eclipse.microprofile.rest.client.tck.RestClientTckPackageTest.testRestClientTck(RestClientTckPackageTest.java:44)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at componenttest.custom.junit.runner.FATRunner$1.evaluate(FATRunner.java:197)
at componenttest.custom.junit.runner.FATRunner$2.evaluate(FATRunner.java:318)
at componenttest.custom.junit.runner.FATRunner.run(FATRunner.java:171)
```
This should be something like:
```
junit.framework.AssertionFailedError: 2021-04-23-16:47:38:906 No TCK test JUnit result files were found in the results directory which suggests the TCK tests did not run
Errors found in mvnOutput were:
### maven test output:
16:47:38,882 [ERROR] Failed to execute goal on project tck.runner.tck: Could not resolve dependencies for project io.openliberty.microprofile.config20.internal:tck.runner.tck:jar:2.0: Failure to find a.dumb.dependency:i-should-break:jar:2.0 in https://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced -> [Help 1]
16:47:38,885 [ERROR]
16:47:38,885 [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
16:47:38,886 [ERROR] Re-run Maven using the -X switch to enable full debug logging.
16:47:38,886 [ERROR]
16:47:38,887 [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
```

--- a/dev/fattest.simplicity/src/componenttest/topology/utils/MvnUtils.java
+++ b/dev/fattest.simplicity/src/componenttest/topology/utils/MvnUtils.java
@@ -194,7 +194,7 @@ public class MvnUtils {
             try (Scanner s = new Scanner(installOutput)) {
                 while (s.hasNextLine() && sb.length() < 20000) {
                     String line = s.nextLine();
-                    if (line.startsWith("[ERROR]"))
+                    if (line.contains("[ERROR]"))
                         sb.append(line).append("\n");
                 }
             } catch (FileNotFoundException e) {
@@ -207,7 +207,7 @@ public class MvnUtils {
             try (Scanner s = new Scanner(testOutput)) {
                 while (s.hasNextLine() && sb.length() < 20000) {
                     String line = s.nextLine();
-                    if (line.startsWith("[ERROR]"))
+                    if (line.contains("[ERROR]"))
                         sb.append(line).append("\n");
                 }
             } catch (FileNotFoundException e) {

